deploy/07_giveaway/18_deploy_multi_giveaways.ts

```diff
@@ -0,0 +1,46 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+
+const multigiveaways = [1, 2];
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts} = hre;
+  const {deploy} = deployments;
+  const {deployer, multiGiveawayAdmin} = await getNamedAccounts();
+
+  const adminByNetwork: {
+    [n: number]: {[name: string]: string | undefined; default: string};
+  } = {
+    1: {
+      default: deployer,
+      mainnet: multiGiveawayAdmin,
+      polygon: multiGiveawayAdmin,
+    },
+    2: {
+      default: deployer,
+      mainnet: '0x2e0405eaC370F83d6A0085D69F77C37dAf8d901F',
+      polygon: '0x44baA6A480401717ed72762958eE2742b47F8b77',
+    },
```

Comment:
Shouldn't this be in the hardhat config?

Reply:
I can put a name on it if that's what you mean, sure